src/modules/reporting/actions/load-reporting-window-bounds.js

```diff
@@ -3,33 +3,19 @@ import { updateReportingWindowStats } from 'modules/reporting/actions/update-rep
 
 export const loadReportingWindowBounds = () => (dispatch, getState) => {
   const { universe, loginAccount } = getState()
-
-  augur.api.Universe.getCurrentFeeWindow({ universe: universe.id }, (err, reportingWindow) => {
-    if (err) return
-
-    augur.api.FeeWindow.getStartTime({ tx: { to: reportingWindow } }, (err, startTime) => {
-      if (err) return
-      dispatch(updateReportingWindowStats({ startTime }))
-    })
-
-    augur.api.FeeWindow.getEndTime({ tx: { to: reportingWindow } }, (err, endTime) => {
+  augur.augurNode.submitRequest(
+    'getFeeWindowCurrent',
+    {
+      universe: universe.id,
+      reporter: loginAccount.address
+    }, (err, result) => {
       if (err) return
```

Comment:
Hm, should there be a callback here (or at least a `console.error`), so if there is an error it propagates to the caller?  Same question for other `if (err) return` lines.

Reply:
Yes we need to at least log it. If error we wint update fee window properties. I’ll fix

Reply:
added callback to log error